sbtc-core/src/signer/mod.rs

```diff
@@ -0,0 +1,119 @@
+/// An sBTC signer configuration
+pub mod config;
+
+use std::collections::HashMap;
+
+use crate::signer::config::Config;
+use crate::SBTCError;
+use bitcoin::{Network, PrivateKey, PublicKey, Transaction as BitcoinTransaction};
+use p256k1::ecdsa;
+use url::Url;
+use wsts::{bip340::SchnorrProof, common::Signature};
+
+#[derive(Default, Clone, Debug)]
+pub struct SignersData {
```

Comment:
Name suggestion: How about just calling this "Signers"?

```suggestion
pub struct Signers {
```